salesapp.js

```diff
@@ -130,10 +216,24 @@ var displaySeaCenter = function () {
   }
   return list;
 };
+
+var totalSeaCntr = function() {
+  var list = document.getElementById('seacenter');
+  var total = 0;
+  for ( var i = 0; i < arraySeattleCntr.length; i++ ){
+    total += arraySeattleCntr[i];
+  }
+  var item = document.createElement('li');
+  item.textContent = 'Total: ' + total + ' cookies sold.';
+  list.appendChild(item);
+  return total;
+};
+
 storeSeattleCntr.avgSales();
 displaySeaCenter();
+totalSeaCntr();
```

Comment:
My advice is to group function calls like this all together at the bottom, that way it's super easy to figure out what the execution order is.